src/Symfony/Component/HttpClient/Chunk/ServerSentEvent.php

```diff
@@ -76,4 +78,30 @@ public function getRetry(): float
     {
         return $this->retry;
     }
+
+    /**
+     * Gets the SSE data decoded as an array when it's a JSON payload.
+     */
+    public function getArrayData(): array
+    {
+        if (null !== $this->jsonData) {
+            return $this->jsonData;
+        }
+
+        if ('' === $this->data) {
+            throw new JsonException(sprintf('Server-Sent Event%sdata is empty.', '' !== $this->id ? sprintf(' "%s" ', $this->id) : ' '));
```

Comment:
`throw new JsonException(sprintf('Server-Sent Event%s data is empty.', '' !== $this->id ? sprintf(' "%s"', $this->id) : ''));`